internal/databases/mongo/oplog/applier.go

```diff
@@ -254,8 +254,8 @@ func indexSpecFromCommitIndexBuilds(op db.Oplog) (string, []client.IndexDocument
 				if !ok {
 					return "", nil, NewTypeAssertionError("bson.D", fmt.Sprintf("indexes[%d]", i), elemE.Value)
 				}
-				for i := range elements {
-					elemE = elements[i]
+				for j := range elements {
```

Comment:
Maybe instead of this line 
'for _, elemE := range elements {'

Reply:
updated